public/docs/_examples/package.json

```diff
@@ -74,6 +74,7 @@
     "raw-loader": "^0.5.1",
     "rimraf": "^2.5.4",
     "rollup-plugin-commonjs": "^4.1.0",
+    "source-map-explorer": "^1.3.2",
```

Comment:
Yeah, the prose talks about how to use it, so I think it makes sense to install it with the sample